### PR TITLE
Add dockerfile and yaml for setting up piecestore servers

### PR DESCRIPTION
**Pull request by aleitner**
This change is [Reviewable](https://reviewable.io/reviews/storj/storj/102)

### CODE REVIEW

File: cmd/piecestore-farmer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# build
+FROM golang:alpine AS build-env
+RUN apk update
+RUN apk upgrade
+RUN apk add gcc musl-dev
+
+ADD . /go/src/storj.io/storj
+RUN cd /go/src/storj.io/storj/cmd/piecestore-farmer && go build -o piecestore-farmer
```
[the remark this answers is not in the document]
Comment (aleitner) [reply cmd/piecestore-farmer/Dockerfile]: vgo still can't install minio

[the remark this answers is not in the document]
Comment (aleitner) [reply cmd/piecestore-farmer/Dockerfile]: also I need gcc and musl-dev for building sqlite

**Comment by coyle** [comment]
Review status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @coyle)

---

*[cmd/piecestore-farmer/Dockerfile, line 8 at r3](https://reviewable.io/reviews/storj/storj/102#-LG6aU2MAhjtNV0dxV3M-r3-8:-LG6pG1g5BuphfPLjb9Z:b-hqd9vd) ([raw file](https://github.com/storj/storj/blob/d9fd046163cc4b6338e22ad6596f1da31568ebba/cmd/piecestore-farmer/Dockerfile#L8)):*
<details><summary><i>Previously, aleitner (Alexander Leitner) wrote…</i></summary><blockquote>

also I need gcc and musl-dev for building sqlite
</blockquote></details>

It can't? that's lame ok, ignore then

---


*Comments from [Reviewable](https://reviewable.io/reviews/storj/storj/102)*

**Comment by aleitner** [comment]
Review status: 0 of 3 files reviewed, 2 unresolved discussions (waiting on @coyle)

---

*[cmd/piecestore-farmer/piecestore-farmer.yaml, line 1 at r3](https://reviewable.io/reviews/storj/storj/102#-LFT476oP0J-Dkj_akKM-r3-1:-LG6qNi9-JZir0Ynx4_o:b-896fix) ([raw file](https://github.com/storj/storj/blob/d9fd046163cc4b6338e22ad6596f1da31568ebba/cmd/piecestore-farmer/piecestore-farmer.yaml#L1)):*
<details><summary><i>Previously, coyle (Dennis Coyle) wrote…</i></summary><blockquote>

We don't need this here any more. These will all be stored in Kubernetes. @brimstone can explain better than I can
</blockquote></details>

Done.

---

*[cmd/piecestore-farmer/piecestore-farmer.yaml, line 38 at r3](https://reviewable.io/reviews/storj/storj/102#-LFT476oP0J-Dkj_akKM-r3-38:-LG6qNQe-RUB3OOslbuJ:b-896fix) ([raw file](https://github.com/storj/storj/blob/d9fd046163cc4b6338e22ad6596f1da31568ebba/cmd/piecestore-farmer/piecestore-farmer.yaml#L38)):*
<details><summary><i>Previously, coyle (Dennis Coyle) wrote…</i></summary><blockquote>

this will be `quay.io/storjlabs/piecestore:master`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/storj/storj/102)*

**Pull request merged**